oarepo_ui/templates/components/datafields/IBaseField.jinja

```diff
@@ -1,4 +1,4 @@
-{#def label, label_class="", data_class="" #}
+{#def label, label_class="ui field-label", data_class="ui field-data" #}
```

Comment:
@Ducica are you sure this works? When someone uses IBaseField and passes custom `label_class` value to it?

Reply:
@mirekys Hello, just tested and it works. 

![image](https://github.com/oarepo/oarepo-ui/assets/34883098/1ba853e9-8f98-4435-82d0-0407ec9770ac)